tests/integration/string/trim-test.js

```diff
@@ -0,0 +1,52 @@
+import { trim } from 'ember-awesome-macros/string';
+import { raw } from 'ember-awesome-macros';
+import { module, test } from 'qunit';
+import compute from 'ember-macro-test-helpers/compute';
+
+module('Integration | Macro | string | trim');
+
+test('returns undefined when doesn\'t exist', function(assert) {
+  compute({
+    assert,
+    computed: trim('source'),
+    strictEqual: undefined
+  });
+});
+
+test('returns undefined when undefined', function(assert) {
+  compute({
+    assert,
+    computed: trim('source'),
+    properties: {
+      source: undefined
+    },
+    strictEqual: undefined
+  });
+});
+
+test('upper cases string', function(assert) {
```

Comment:
I assume the description here is a copy/paste error? Should probably be `trims string`.

Reply:
You're right just basd copy paste.